Glyssen/Bundle/GlyssenDblTextMetadata.cs

```diff
@@ -391,19 +391,26 @@ public enum PreferenceLevel
 		Required,
 	}
 
+	[Flags]
 	public enum ExtraBiblicalMaterialSpeakerOption
 	{
-		Narrator,
-		ActorOfEitherGender,
-		MaleActor,
-		FemaleActor,
-		Omitted,
+		Narrator = 1 << 0,
+		ActorOfEitherGender = 1 << 1,
+		MaleActor = 1 << 2,
+		FemaleActor = 1 << 3,
+		Omitted = 1 << 4,
+		MalePermitted = MaleActor | ActorOfEitherGender,
+		FemalePermitted = FemaleActor | ActorOfEitherGender,
```

Comment:
MaklePermitted and FemalePermitted are not used. Are they there for some future (planned) use?